src/main/scala/com/cognite/sdk/scala/v1/fdm/common/properties/PropertyType.scala

```diff
@@ -86,6 +81,8 @@ object PropertyType {
 
   implicit val externalIdReferenceEncoder: Encoder[CDFExternalIdReference] =
     Encoder.forProduct1("type")(_.`type`)
+  // TODO: Change this to below once list types are released
+  //    Encoder.forProduct2("type", "list")(t => (t.`type`, t.list))
```

Comment:
`TimeSeriesReference`, `FileReference` & `SequenceReference` will get `list` types in the future
Added some `TODO`s to make it easy